Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.16.0-alpine
+
+WORKDIR /workspace
+
+COPY package*.json ./
```

Comment:
Do we need to copy here, `/workspace`?

Reply:
We could specify the work directory name but don't have to. `./` refers to the current workdir: https://docs.docker.com/engine/reference/builder/#copy

The advantage of using `./` instead of `/workspace` is that if we ever want to rename the work directory it's a one line change